cluster/restore-from-backup.sh

```diff
@@ -103,13 +103,13 @@ wait_for_cluster_healthy() {
 
 # Wait until etcd and apiserver pods are down.
 wait_for_etcd_and_apiserver_down() {
-  for i in $(seq 120); do
-    etcd=$(docker ps | grep etcd-server | wc -l)
-    apiserver=$(docker ps | grep apiserver | wc -l)
+  for _ in $(seq 120); do
+    etcd=$(docker ps | grep -c etcd-server)
+    apiserver=$(docker ps | grep -c apiserver)
     # TODO: Theoretically it is possible, that apiserver and or etcd
     # are currently down, but Kubelet is now restarting them and they
     # will reappear again. We should avoid it.
-    if [ "${etcd}" -eq "0" -a "${apiserver}" -eq "0" ]; then
+    if [ "${etcd}" -eq "0" ] && [ "${apiserver}" -eq "0" ]; then
```

Comment:
the test command is unspecified in POSIX with > 4 arguments. it's probably fine (?) in practice in bash even though we're using the posix test. but this is better & functionally equivilant.

this is the minimally equivalent-yet-more-correct form